app/seqexec-server-gs/src/main/resources/app.conf

```diff
@@ -62,7 +62,7 @@ seqexec-engine {
   ioTimeout = 4 seconds
   dhsTimeout = 30 seconds
   gpiUrl = "failover:(tcp://172.17.107.50:61616)?timeout=4000"
-  gpiGDS = "http://172.17.107.50:8888/xmlrpc"
```

Comment:
Just out of curiosity, Does this imply a change in the gpisimulator.cpp and gpiobssimulator.cpp examples of the giapi-glue-c++?